src/current/v23.1/import-performance-best-practices.md

```diff
@@ -7,61 +7,24 @@ docs_area: migrate
 
 This page provides best practices for optimizing [import](import-into.html) performance in CockroachDB.
 
-Import speed primarily depends on the amount of data that you want to import. However, there are two main factors that have can have a large impact on the amount of time it will take to run an import:
+Import speed primarily depends on the amount of data that you want to import. However, there are three actions you can take before importing that have a significant impact on the amount of time it will take to run an import:
```

Comment:
Do we have a convention on IMPORT or Import?

Reply:
Looks like we use "import" and "IMPORT INTO" interchangably. I will make an editing pass to make this clear.